### PR TITLE
[games-action/supertuxkart] version bump to 0.9

### PR DESCRIPTION
**Pull request by nado**
Hi, my first ebuild actually working.
Didnt try the useflags (will test USE="debug" soon).

### CODE REVIEW

**Comment by hasufell** [comment]
trying it out now

**Comment by hasufell** [comment]
I can't seem to fetch it. Maybe the tarballs has not propagated through the sourceforge mirrors yet?

**Comment by nado** [comment]
Oh, right sorry I didnt tell you, the tarball has been available only temporary for test purposes, it will be put online again tomorrow the later.
Im uploading the version i got here : http://www.troglodyte.be/files/supertuxkart-0.9.tar.xz (in ~2hours, 50kB/s upload)
Since an official one will be available tomorrow, your call if you wanna check the ebuild this evening or tomorrow.

File: games-action/supertuxkart/metadata.xml [hunk 1]
```diff
@@ -5,6 +5,9 @@
 		<email>hasufell@posteo.de</email>
 		<name>Julian Ospald</name>
 	</maintainer>
+	<maintainer>
+		<email>nado@troglodyte.be</email>
+		<name>Corentin Pazdera</name>
```
Comment (hasufell) [inline games-action/supertuxkart/metadata.xml]: here was the closing tag missing, I fixed it in 48d0a4c0a4eff183fbe339262ef5207051efe7e8

**Comment by hasufell** [comment]
afais we can even drop the `media-libs/freeglut` dep, can we?

**Pull request merged**